<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCardDialog</class>
 <widget class="QDialog" name="AddCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Card</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_nombre">
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nombreLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_apellido">
     <property name="text">
      <string>Apellido:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="apellidoLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_dni">
     <property name="text">
      <string>DNI:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="dniLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_patente">
     <property name="text">
      <string>Patente:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="patenteLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_color">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="colorLineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
